packages/fscomponents/src/components/MultiCarousel/MultiCarousel.web.tsx

```diff
@@ -255,11 +255,16 @@ export const MultiCarousel = <ItemT, >(props: MultiCarouselProps<ItemT>) => {
       if (scrollViewElement instanceof HTMLElement && !animating) {
         setAnimating(true);
         await animatedScrollTo(scrollViewElement, nextIndex * pageWidth, animated ? 200 : 0)
-          .then(() => setAnimating(false))
-          .catch(e => {
-            setAnimating(false);
-            console.warn('animatedScrollTo error', e);
-          });
+        .then(() => setAnimating(false))
+        .catch(e => {
+          setAnimating(false);
+          console.warn('animatedScrollTo error', e);
+        });
```

Comment:
The changes to the formatting here are unnecessary

Reply:
Indeed. Amended the formatting.